tests/e2e/tour/tour.spec.js

```diff
@@ -9,10 +11,93 @@ const expectWithinMargin = async (actual, expected, margin) => {
   })
 }
 
+test.describe('When clicking the "Skip for now" button', () => {
+  test.beforeEach(async ({ page, context }) => {
```

Comment:
This exact block is in this file 4 times. To reduce the repetition add a top level wrapping test.describe() and put this beforeEach within that new block, so every test.describe() block can reuse it